lua/ulx/modules/sh/cfc_stroke.lua

```diff
@@ -0,0 +1,137 @@
+    function( word )
+        -- Add "y" to the end of the word
+        return word .. "y"
+    end,
+    function( word )
+        -- Repeat last letter
+        return word .. string.sub( word, -1 )
+    end,
+    function( word )
+        -- Remove last letter
+        return string.sub( word, 1, -2 )
+    end,
+    function( word )
+        -- Add "ing" to the end of the word
+        return word .. "ing"
+    end,
+    function( word )
+        -- Duplicate random letter
+        local rand = math.random( 1, #word )
+        local ind = math.random( 1, #word - 1 )
+        return string.sub( word, 1, ind ) .. string.sub( word, rand, rand ) .. string.sub( word, ind + 1 )
+    end,
+    function( word )
+        -- Remove vowels
+        word = string.gsub( word, "[AEIOUaeiou]", "" )
+        return word
+    end,
+    function( word )
+        -- Remove word
+        return ""
+    end,
+    function( word )
+        -- Uppercase
+        return string.upper( word )
+    end,
+    function( word )
+        -- Repeat word
+        return word .. " " .. word
+    end,
+    function( word )
+        -- Repeat word elsewhere in the sentence
+        return word, word
+    end,
+    function( word )
+        -- Replace word with "um" or "uh"
+        return math.random() < 0.5 and "um" or "uh"
+    end,
+    function( word )
+        -- Insert a question mark at the end of the word
+        return word .. "?"
+    end,
+    function( word )
+        -- Remove a random letter
+        local rand = math.random( 1, #word )
+        return string.sub( word, 1, rand - 1 ) .. string.sub( word, rand + 1 )
+    end,
+}
+
+local wordStrokeChance = 0.5
+
+
+local function getWordTransformation()
+    return wordTransformations[math.random( 1, #wordTransformations )]
+end
+
+local function transform( sentence )
+    local transformedWords = {}
+    local words = string.Split( sentence, " " )
+
+    while #words ~= 0 do
+        local word = table.remove( words, 1 )
+        local shouldStroke = math.random() < wordStrokeChance
+
+        if shouldStroke then
+            local transformationResult = { getWordTransformation()( word, transformedWords ) }
+            local transformedWord = table.remove( transformationResult, 1 )
+
+            if transformedWord ~= "" then
+                table.insert( transformedWords, transformedWord )
+            end
+
+            for _, additionalWord in ipairs( transformationResult ) do
+                -- Insert additional words at a random position in the original sentence
+                local rand = math.random( 1, #words + 1 )
+                table.insert( words, rand, additionalWord )
+            end
+        else
+            table.insert( transformedWords, word )
+        end
+    end
+
+    -- Shuffle a random amount of words
+    for i = 1, math.random( 0, math.floor( #transformedWords / 2 ) ) do
```

Comment:
```suggestion
    for _ = 1, math.random( 0, math.floor( #transformedWords / 2 ) ) do
```

Reply:
Thanks Straw, I'll push those fixes. I based my boilerplate off of zoomer and I'm a little rusty.